test/test.spec.js

```diff
@@ -0,0 +1,46 @@
+
+var mocha  = require('mocha');
+var stackFilter = mocha.utils.stackTraceFilter();
+var assert = require('chai').assert;
+var expect = require('chai').expect;
+var VError = require("verror");
+var MultiError = VError.MultiError;
+var WError = VError.WError;
+var MochaVerrorReporter = require('../built/mochaVErrorReporter.js').MochaVErrorReporter;
+var fullStack = MochaVerrorReporter.fullStack;
+
+describe("MochaVErrorReporter", () => 
+{
+    var indentChars = "    ";
+    var indentStart = 1;
+    var sampleError = new MultiError(
+        [
+            new WError(
+                new MultiError(
+                    [
+                        new WError(
+                            new WError("a nested error inside a nested error"),
+                            "a nested error"
+                        ),
+                        new Error("another nested error")
+                    ]
+                ),
+                "encapsulating a multierror"
+            ),
+            new WError("exists at the same level with encapsulator")
+        ]
+    );
+
+    var fnOutput = fullStack(sampleError, indentChars, indentStart).replace(/\n.*\s{4}at\s.*/g, "");
+    
+    it("recursively outputs errors with correct indentation", function()
+    {
+        const expectedOutput =  "MultiError: first of 2 errors: encapsulating a multierror\n" +
```

Comment:
Is it possible not to repeat ``encapsulating a multierror`` here. 
Can we format like the following:

    MultiError: with 2 suppressed errors
        error #1/2: WError: encapsulating a multierror
            caused by, MultiError: with 2 suppressed errors
                error #1/2:: WError: a nested error
                     caused by, WError: a nested error inside a nested error
                error #2/2:: Error: another nested error
        error #2/2: WError: exists at the same level with encapsulator

Reply:
Fixed